src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -2129,4 +2129,13 @@ Widening and user defined conversions are not supported with generic types.</val
   <data name="UseStringMethodCharOverloadWithSingleCharactersTitle" xml:space="preserve">
     <value>Use char overload</value>
   </data>
+  <data name="AvoidSingleUseOfLocalJsonSerializerOptionsMessage" xml:space="preserve">
+    <value>Avoid single-use of local 'JsonSerializerOptions' as it is suboptimal, use a singleton instance instead to improve performance</value>
```

Comment:
Looks the message could use some improvement, does below sound better? 
```suggestion
    <value>Prefer reusing 'JsonSerializerOptions' instance by moving the local instance to static field or use singleton through DI</value>
```
Tag @gewarren for more improvement.

Reply:
I just don't feel like `single-use of local` is important here, please disregard the comment if I am wrong.

Reply:
```suggestion
    <value>Reuse 'JsonSerializerOptions' instance by moving it to a static field, or use a singleton</value>
```

Reply:
From https://github.com/dotnet/roslyn-analyzers/pull/6850#discussion_r1291667219:

> Single-use 'JsonSerializerOptions' instances can substantially degrade the performance of your applicaton, consider using a singleton instead.

Both suggestions sound fine to me.